__tests__/app.test.js

```diff
@@ -150,6 +150,50 @@ describe("/api/reviews/:review_id/comments", ()=> {
     expect(res.body).toEqual({msg:"Error - review ID not found"})
 })
 })
+test("POST - status 201 - responds with the added comment object", () => {
+  return request(app)
+  .post("/api/reviews/8/comments")
+  .expect(201)
+  .send({username:'efie', body:'great game'})
+  .then((result => {
+    
+    const {comment}= result.body
+    expect(comment.body).toBe('great game')
+    expect(comment.author).toBe("mallionaire")
+    expect(comment.votes).toBe(0)
+    expect(comment.review_id).toBe(8)
+    expect(typeof comment.comment_id).toBe('number')
+    expect(typeof comment.created_at).toBe("string")
+
+  }))
+})
+test ("POST - status 400 - review ID not of correct form returns bad request error", () => {
+  return request(app)
+  .post("/api/reviews/nonsense/comments")
+  .expect(400)
+  .send({username:'efie', body:'great game'})
+  .then((res) => {
+  expect(res.body).toEqual({msg:"Error - bad request!"})
+})
+})
+test ("POST - status 400 - if property is missing from the comment object in the request", () => {
+  return request(app)
+  .post("/api/reviews/3/comments")
+  .expect(400)
+  .send({})
+  .then((res) => {
+  expect(res.body).toEqual({msg:"Error - bad request, comment missing properties!"})
+})
+})
+test("POST - status 404 - invalid review ID returns ID not found error", ()=> {
+  return request(app)
+  .post("/api/reviews/3000/comments")
+  .expect(404)
+  .send({username:'efie', body:'great game'})
+  .then((res) => {
+  expect(res.body).toEqual({msg:"Error - review ID not found"})
+})
+})
```

Comment:
A couple more tests I'd like to see:
What if the username doesn't exist in the database?
What if additional properties are on the comment object?